{
  "Header": {
    "Original": "CVE-2018-10872 (A flaw was found in the way the Linux kernel handled exceptions delive ...)",
    "Line": 343639,
    "ID": "CVE-2018-10872",
    "Description": "(A flaw was found in the way the Linux kernel handled exceptions delive ...)"
  },
  "Annotations": [
    {
      "Original": "- linux \u003cnot-affected\u003e (Red Hat specific CVE-2018-8897 regression in RHEL 6.10)",
      "Line": 343640,
      "Type": "package",
      "Package": "linux",
      "Kind": "not-affected",
      "Description": "Red Hat specific CVE-2018-8897 regression in RHEL 6.10"
    },
    {
      "Original": "NOTE: https://bugzilla.redhat.com/show_bug.cgi?id=1596094",
      "Line": 343641,
      "Type": "NOTE",
      "Description": "https://bugzilla.redhat.com/show_bug.cgi?id=1596094"
    }
  ]
}